pkg/operator/configobservation/etcd/observe_etcd.go

```diff
@@ -139,7 +139,7 @@ func ObserveClusterMembers(genericListers configobserver.Listers, recorder event
 func ObservePendingClusterMembers(genericListers configobserver.Listers, recorder events.Recorder, currentConfig map[string]interface{}) (observedConfig map[string]interface{}, errs []error) {
 	listers := genericListers.(configobservation.Listers)
 	observedConfig = map[string]interface{}{}
-	clusterMemberPath := []string{"cluster", "pending"}
+	clusterMemberPath := []string{"cluster", "members"}
 
```

Comment:
what is the theory here?

Reply:
removed the commit. Thanks for the catch